rudder-webapp/SOURCES/Makefile

```diff
@@ -19,6 +19,17 @@
 MAVEN_RELEASE = 3.0.4
 WGET := $(if $(PROXY), http_proxy=$(PROXY) ftp_proxy=$(PROXY)) /usr/bin/wget -q
 
+MAVEN_PROXY := $(if $(PROXY), $(shell echo "$(PROXY)" |  perl -ne ' \
+    if (m|(https?)://(?:([^:@]+)(?::([^@]+))?\@)?([^:/]+)(?::([0-9]+))?|) { \
```

Comment:
'(?:' means do not remember this field, so the $x variables where x>=2 will be empty